microcosm_logging/factories.py

```diff
@@ -187,12 +188,14 @@ def make_loggly_handler(graph, formatter):
 
     """
     base_url = graph.config.logging.loggly.base_url
+    metric_service_name = environ.get("METRICS_NAME", graph.metadata.name)
     loggly_url = "{}/inputs/{}/tag/{}".format(
         base_url,
         graph.config.logging.loggly.token,
         ",".join([
             graph.metadata.name,
             graph.config.logging.loggly.environment,
+            metric_service_name
```

Comment:
Trailing comma.

Also, are you going to dedupe this (in case metrics_service_name == graph.metadata.name).

Reply:
I tested with the metric_service_name being the same as the metadata.name and the duplicate tag was just ignored in loggly and only one tag was displayed.

Reply:
Probably true, but it's a bit misleading from the code side. Someone else will probably notice the same thing (duplicate tags) and wonder if we're duping all the way into loggly.

Reply:
Yes. Made changes in the last commit.